xwiki-platform-core/xwiki-platform-mentions/pom.xml

```diff
@@ -43,12 +43,49 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jetbrains</groupId>
+        <artifactId>annotations</artifactId>
+        <version>19.0.0</version>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
   <modules>
     <module>xwiki-platform-mentions-api</module>
     <module>xwiki-platform-mentions-notifications</module>
     <module>xwiki-platform-mentions-ui</module>
     <module>xwiki-platform-mentions-default</module>
   </modules>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.revapi</groupId>
+        <artifactId>revapi-maven-plugin</artifactId>
+        <configuration>
+          <analysisConfiguration><![CDATA[
```

Comment:
Revapi ignores are put in xwiki-platform-core/pom.xml so they can be processed all at once during release. And I think the new format for them is XML but maybe it changed again lately.

Reply:
Yes it's XML and that's also what revapi suggest you in the error AFAIK.